src/github.com/getlantern/flashlight/app/settings.go

```diff
@@ -30,7 +28,7 @@ var (
 // Settings is a struct of all settings unique to this particular Lantern instance.
 type Settings struct {
 	DeviceID  string `json:"deviceID,omitempty"`
-	UserID    int    `json:"userID,omitempty"`
+	UserID    string `json:"userID,omitempty"`
```

Comment:
Just use a string instead of an int since the int makes use do so many convoluted things for little to no gain.